orangecontrib/spectroscopy/preprocess/__init__.py

```diff
@@ -758,6 +758,40 @@ def __call__(self, data):
         return data.from_table(domain, data)
 
 
+class _PhaseUnwrapCommon(CommonDomain):
+
+    def __init__(self, unwrap, domain):
+        super().__init__(domain)
+        self.unwrap = unwrap
+
+    def transformed(self, data):
+        return np.unwrap(data.X)
+
+
+class PhaseUnwrap(Preprocess):
+    """
+    Unwrap phase values using numpy.unwrap defaults or bypass the data.
+
+    Parameters
+    ----------
+    unwrap    : toggle unwrap/bypass (boolean)
+    """
+
+    def __init__(self, unwrap=True):
```

Comment:
Just a curiosity: why does this preprocessor has a bypass switch?

If this for testing or do you have a real usecase?

Reply:
Actually, i don't quite remember why, i think i was inspired by another widget.